examples/meteor/client/styles/base.css

```diff
@@ -285,9 +284,9 @@ label[for='toggle-all'] {
 	display: block;
 }
 
-#todo-list li .edit {
-	display: none;
-}
+/*#todo-list li .edit {*/
+	/*display: none;*/
+/*}*/
```

Comment:
And what exactly is the deal with this? We should consider this as a vendor file and it shouldn't be changed. I don't like the fact that it's not fetched using a package manager anymore but just pulled in as a regular style eighter (or is it?)